db/migrate/20160928080819_copy_rearranged_uuids_to_transactions.rb

```diff
@@ -0,0 +1,10 @@
+class CopyRearrangedUuidsToTransactions < ActiveRecord::Migration
+  def up
+    execute "UPDATE transactions, listings SET transactions.listing_uuid = listings.uuid WHERE transactions.listing_id = listings.id"
+    execute "UPDATE transactions, communities SET transactions.community_uuid = communities.uuid WHERE transactions.community_id = communities.id"
+  end
+
+  def down
+    execute "UPDATE transactions SET listing_uuid = NULL, community_uuid = NULL"
+  end
```

Comment:
NULL is not allowed for `listing_uuid` and `community_uuid`.

I'd suggest you just remove the `down` migration. The original state where we were before running the migrations is something we can't go back to, because we overrode the original values.